spec/core/ics-026-routing-module/README.md

```diff
@@ -432,16 +430,14 @@ function handleChanOpenInit(datagram: ChanOpenInit) {
     datagram.order,
     datagram.connectionHops,
     datagram.portIdentifier,
-    datagram.channelIdentifier,
```

Comment:
This was also not correct here since the channel identifier is generated in INIT by the handler and it's not in the datagram.